rl_loop.py

```diff
@@ -128,10 +128,23 @@ def train(logdir=None):
         logging.exception("Train error")
 
 
+def validate(logdir=None, model_num=None):
+    if model_num is None:
+        model_num, model_name = get_latest_model()
+    else:
+        model_name = get_model(model_num)
+
+    holdout_set = get_model(model_num - 1)
```

Comment:
model_num - 1? That's a little counter intuitive. Add a comment?

Reply:
I think validation should probably run on last 50 generations (or maybe even ALL the generations) because we want to verify that our ability to predict game outcomes isn't degrading. So we'd want to go back and look at really old games too, not just the most recent one.

Reply:
So we would be checking a new model against the holdout data of the previous N models?  that sounds fine.  That makes my comments about filter_amount above irrelevant, and i'll revert it to 5%

Reply:
Are you thinking 1 plot per generation (that's a lot of plots)? Or 1 plot that has generation id on the x-axis and measures the error on this generation's self play data/pro data?

Reply:
@brilee this would make it share a lot of the functionality i was noodling with on my [train from n branch](https://github.com/amj/minigo/blob/train-from-n/main.py#L89), as far as filtering the training chunks by their model name.  I could do something like that here as well.